### DISCUSSION TITLE
Different Styling for Different Lock Reasons (Booked vs. Unavailable due to Min Days)

### DISCUSSION THREAD

**Discussion by darren-he** [easepick/easepick#222] [Q&A]
Using the lock-plugin, I'm able to disable certain dates (in the context of a hotel-style date range picker).

However, is it possible to have different styling to visually differentiate dates that are locked due to being "booked", vs. dates that are locked due to being less than minDays?

It seems that the "locked" class is what is used to show dates on the calendar as blocked. Anyone figure out a way to further differentiate? Thanks.

**Comment by fredasterisk** [comment]
Does the [Hotel example](https://easepick.com/examples/hotel.html) look something like what you're looking to achieve?
If so, they are styling "not-available" (minDates) and "locked" (booked) classes using an external CSS file that you can add through the css: [] parameters.
